src/taipy/rest/api/resources/cycle.py

```diff
@@ -49,17 +52,20 @@ class CycleResource(Resource):
                 properties:
                   cycle: CycleSchema
         404:
-          description: cycle does not exist
+          description: Cycle does not exist
     delete:
       tags:
         - api
       summary: Delete a cycle
-      description: Delete a single cycle by ID
+      description: |
+        Delete a single cycle by CycleId. If the cycle does not exist, a 404 error is returned.
```

Comment:
"Delete a single cycle by CycleId"
->
"Deletes the cycle identified by *cycle_id*."